modules/server/src/test/scala/higherkindness/compendium/http/RootServiceSpec.scala

```diff
@@ -93,6 +95,23 @@ object RootServiceSpec extends Specification with ScalaCheck {
         .map(_.value) === Some(s"/protocol/$id")
     }.setGen(Gen.alphaNumStr suchThat (!_.isEmpty))
 
+    "If json is invalid returns a 422 Unprocessable entity" >> prop { id: String =>
```

Comment:
Test description is wrong
```suggestion
    "If json is invalid returns a 400 Bad Request" >> prop { id: String =>
```